scripts/create-example-tests.js

```diff
@@ -97,6 +98,9 @@ const createExampleTests = async ({ directory, args = {} }) => {
   const testPlanBuildDirectory = path.join(buildDirectory, directory);
   const indexFileBuildOutputPath = path.join(testPlanBuildDirectory, 'index.html');
 
+  // create build directory if it doesn't exist
+  fs.existsSync(buildDirectory) || fs.mkdirSync(buildDirectory);
```

Comment:
I think we can use the `recursive` option here and remove the `existsSync` call.

```js
fs.mkdirSync(buildDirectory, {recursive: true});
```

https://nodejs.org/dist/latest-v16.x/docs/api/fs.html#fsmkdirsyncpath-options

Reply:
Pushed the recommended change @mzgoddard